### ISSUE TITLE
Json Schema Validation is currently invalid

### ISSUE THREAD

**Issue by ferdynator** [localcosmos/localcosmos-server#8]
Creating an ObservationForm is currently not possible because the `CamelCaseJSONParser` parser is executed before the serializer and validator. This means that any fields the validator sees are transformed to `snake_case` while the current Json Schema clearly states `camelCase`for the form field definitions.

**Comment by ferdynator** [comment]
To solve this add the following to the REST_FRAMEWORK configuration:

```
'JSON_UNDERSCOREIZE': {
    'no_underscore_before_number': True,
    'ignore_fields': ('definition', 'data',),
},
```

Additionally the schema currently states that the `step` widgetAttribute is expected to be a number while the app-kit outputs it as a string.

**Comment by biodiv** [comment]
implemented in `v13.7`
